include/aspect/material_model/reaction_model/katz2003_mantle_melting.h

```diff
@@ -89,11 +89,47 @@ namespace aspect
                           const double pressure,
                           const double maximum_melt_fraction,
                           const NonlinearDependence::Dependence dependence) const;
+
+
+          /**
+           * Compute all the reaction rate variables needed for a reactive transport model based on the
+           * Katz 2003 formulation. Takes @p model inputs and @p model outputs to return.
+          */
+          void calculate_reaction_rate_outputs(const typename Interface<dim>::MaterialModelInputs &in,
+                                               typename Interface<dim>::MaterialModelOutputs &out) const;
+
+          /**
+           * Compute all the fluid variables needed for a reactive transport model based on the
+           * Katz 2003 formulation. Takes @p model inputs and @p model outputs to return.
+          */
+          void calculate_fluid_outputs(const typename Interface<dim>::MaterialModelInputs &in,
+                                       typename Interface<dim>::MaterialModelOutputs &out,
+                                       const double thermal_expansivity,
```

Comment:
Is `thermal_expansivity` above and `reference_T` below properties of the fluid? If yes, it might be worth adding `_fluid` for clarity.

Reply:
or `_melt`

Reply:
I think those are both solid properties. `thermal_expansivity` is the solid thermal expansion coefficient and `reference_T` is the reference temperature used to calculate the solid viscosity temp dependence.